faq.html

```diff
@@ -136,7 +137,7 @@ <h2 class="topic text-center">Git Related Questions</h2>
             <div class="faq">
               <div class="faq__title">
                 <h3 class="question">
-                  I don't know Git. Can I still contribute?
+                  <a class = "user_FAQ" href = "#I-dont't-know-git-can-I-still-contribute" style = "color: black;">I don't know Git. Can I still contribute?</a>
```

Comment:
` style = "color: black;"`, here we are explicitly adding style and are repeating the same everywhere, could we look into the way where we can add in user_FAQ class itself

Reply:
okay so You want me to add this style in user_FAQ class itself rather then repeating?

Reply:
Could you try and check whether it would be helpful and ease the process?
And let me know which one would be better

Reply:
Styling things in the class is obviously better than repeating same thing again and again.